docs/community/project-showcase.md

```diff
@@ -33,4 +54,11 @@ The first example of Monty moving its sensors in the real-world.
 
 See the [everything_is_awesome repository](https://github.com/thousandbrainsproject/everything_is_awesome) for more information.
 
+## Ultrasound Perception
+
+![](../figures/community/ultrasound_robot.jpg)
+
+Using sensorimotor AI to guide ultrasound.
+
+See the [ultrasound_perception repository](https://github.com/thousandbrainsproject/ultrasound_perception) for more information.
```

Comment:
We also need to update this page & can cross link to it here https://thousandbrainsproject.readme.io/docs/using-monty-for-robotics#example-2-ultrasound

Reply:
Can you also add a link to here https://thousandbrainsproject.readme.io/docs/using-monty-for-robotics#example-3-lego-based-robot in the Lego section?

Reply:
I don't see these added yet. Am I looking at the wrong view?

Reply:
ah, I misunderstood, I only did the back link.